doc/_static/draw_mpl/draw_mpl_examples.py

```diff
@@ -97,7 +97,7 @@ def rcparams(circuit):
 
     plt.savefig(folder / "rcparams.png")
     plt.close()
-    plt.style.use("default")
+    qml.drawer.use_style("black_white")
```

Comment:
All of the functions called after `rcparams` are affected by the updates to `plt.rcParams`, which causes the lines to be bolder in `wires_labels.png`, `mid_measure.png` in the `doc/_static/draw_mpl` directory.

Reply:
Images just had to be re-rendered.